openshift-environment-driver/pom.xml

```diff
@@ -54,6 +54,10 @@
       <groupId>org.jboss.pnc</groupId>
       <artifactId>spi</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.jboss.pnc</groupId>
+      <artifactId>build-coordinator</artifactId>
+    </dependency>
 
```

Comment:
An option is to make it configurable from the code before the first usage.
But being part of common, where should be no PNC specific stuff, I think it is ok to configure it via system property. To keep all the configurations in the pnc-config.json, we can set the system property programatically on config load.
Looking again at the added dependency, we cannot do that, as build-coordinator should be decoupled from the executor (environment driver and other). The plan is to extract executor as its own service.